src/cli/commands/lint.ts

```diff
@@ -105,6 +105,11 @@ const lintCommand: CommandModule = {
           type: 'boolean',
           default: false,
         },
+        'ignore-unmatched-globs': {
+          description: 'do not warn about unmatched glob patterns',
+          type: 'boolean',
+          default: true,
```

Comment:
@P0lip In the spirit of #678 could we make this default to `false`?

The idea being to provide the user with meaningful messages by default.

Reply:
We _could_, but this may result in some perf hit, as we might need to visit several directories multiple times.
Might be a not big deal, though. I'll consider doing it before we merge the PR.

Reply:
Alternatively, we could rename the option to `show-unmatched-globs`. What do you think?

Reply:
@nulltoken bump

Reply:
@P0lip Sorry. I overlooked this one :(